examples/common/async_asyncio_client.py

```diff
@@ -174,9 +169,11 @@ def start_loop(loop):
     t.daemon = True
     # Start the loop
     t.start()
+    time.sleep(1)
```

Comment:
If this is a coroutine, `asyncio.sleep()` may be better

Reply:
done